### ISSUE TITLE
Option to specify desired read coverage or sequencing depth

### ISSUE THREAD

**Issue by HLHsieh** [bcgsc/NanoSim#188]
Hi there, 

I am wondering whether there is any option for specifying desired read coverage and sequencing depth for simulation. I only saw the option to specify the exact number of reads for simulation.

Please advise.

Thanks,
Hsin

**Comment by kmnip** [comment]
No, there is no feature for specifying read depth because that kind of information is not tracked during simulation.

**Comment by HLHsieh** [comment]
Hi @kmnip, 

Thanks for your prompt reply. I also had this similar issue as #76. Any advice is greatly appreciated.

Many thanks,
Hsin
